website/content/docs/agent/limits/index.mdx

```diff
@@ -29,8 +29,12 @@ Setting limits per source IP requires Consul Enterprise.
 
 ### Order of operations
 
-You can define request rate limits in the agent configuration and in the control plane request limit configuration entry. The configuration entry also supports rate limit configurations for Consul resources. Consul perfroms the following order of operations when determing request rate limits:
+You can define request rate limits in the agent configuration and in the control plane request limit configuration entry. The configuration entry also supports rate limit configurations for Consul resources. Consul performs the following order of operations when determing request rate limits:
 
+![Diagram showing the order of operations for determining request rate limits.](/img/agent-rate-limiting-ops-order.jpg#light-theme-only)
+![Diagram showing the order of operations for determining request rate limits.](/img/agent-rate-limiting-ops-order-dark.jpg#dark-theme-only)
```

Comment:
i think the dark mode coloring is kinda off?

<img width="733" alt="image" src="https://github.com/hashicorp/consul/assets/7556325/bd104591-d72e-4670-83b6-c28c1560a8f0">

Reply:
I think I was actually using the wrong the slides --- we don't have the ability to do portrait, so I made a copy (of the wrong slides) and followed the instructions there.